### PR TITLE
[CI][Buildkite] An example test for Buildkite

### PR DESCRIPTION
**Pull request by kevin85421**
## Why are these changes needed?

The Buildkite infrastructure for KubeRay CI is ready (Thanks @krfricke!). This PR is used to test the infrastructure after it is merged. 

## Related issue number
#695

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

```bash
docker run -it golang:1.17

# Execute the following commands in the container.
git clone https://github.com/ray-project/kuberay.git
cd kuberay/ray-operator
make test
```

### CODE REVIEW

**Comment by architkulkarni** [comment]
Nice, should I be able to see the test somewhere here?
Screenshot 2023-02-21 at 3 42 32 PM

File: .buildkite/example.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- label: Buildkite example
+  instance_size: large
+  image: golang:1.17
+  commands:
+    - ls -alp /workdir
+    - cd /workdir/kuberay/ray-operator
```
Comment (kevin85421) [inline .buildkite/example.yml:6]:
```suggestion
    - cd /workdir/ray-operator
```

Comment (krfricke) [reply .buildkite/example.yml:6]: I think this should be `/workdir/ray-operator`

**Review by krfricke — APPROVED** [review]
I think the directory should be `/workdir/ray-operator`.

Let's merge this and I'm happy to kick off a test run

**Pull request merged**